### DISCUSSION TITLE
Mac OS 12.6.5 no output from Motu M4 to server

### DISCUSSION THREAD

**Discussion by Rhythmking1** [jamulussoftware/jamulus#3396] [Bug Investigation]
I habv a MacBook Pro 3.1GHz i5 running OS12.6.5. Installed Jamulus 3.11.0 and there is no output from my Motu M4 to the server. I can hear the drums locally but nothing broadcasts. There was no problem before installing this recent upgrade. Is there an incompatibility issue that can be solved. Thanks.

**Comment by ann0see** [comment]
Hi @Rhythmking1

Could you please check if the same issue occurs with internal speakers and microphone?
If not, then check if there's any issue with the Motu M4 (muted inputs and only local monitoring?).

Otherwise, check if Jamulus has permissions to access the microphone (Check via System preferences).
Also, since the bundle ID changed, check if the correct in/out devices are selected in Jamulus' settings.

**Comment by Rhythmking1** [comment]
I will do that but you should know that I am NOT having the same problem on my newer MacBook Pro with recent OS….so I fear it is the older OS that may be the issue only with the update.  This did not occur with the older MacBook before the upgrade as I have been using it for over a year.DavidDavid B Vickar  BSc MD FRCPC FACR ***@***.*** Oct 3, 2024, at 12:19, ann0see ***@***.***> wrote:﻿
Hi @Rhythmking1
Could you please check if the same issue occurs with internal speakers and microphone?
If not, then check if there's any issue with the Motu M4 (muted inputs?).
Otherwise, check if Jamulus has permissions to access the microphone (Check via System preferences).

**Comment by ann0see** [comment]
You could also try the legacy version of Jamulus (mac_legacy), but this shouldn't be required.

https://github.com/jamulussoftware/jamulus/releases/tag/r3_11_0
